modules/node-shared/src/main/scala/org/tessellation/node/shared/http/routes/ClusterRoutes.scala

```diff
@@ -48,13 +48,11 @@ final case class ClusterRoutes[F[_]: Async](
           .join(peerToJoin)
           .flatMap(_ => Ok())
           .recoverWith {
-            case NodeStateDoesNotAllowForJoining(nodeState) =>
-              Conflict(s"Node state=${nodeState} does not allow for joining the cluster.")
+            case NodeStateDoesNotAllowForJoining(nodeState) => Conflict(s"Node state=$nodeState does not allow for joining the cluster.")
             case PeerAlreadyJoinedWithNewerSession(id, _, _, _) => Conflict(s"Peer id=${id.show} already joined with newer session.")
-            case SessionAlreadyExists =>
-              Conflict(s"Session already exists.")
-            case _ =>
-              InternalServerError("Unknown error.")
+            case SessionAlreadyExists                           => Conflict(s"Session already exists.")
+            case PeerNotInSeedlist(id)                          => Conflict(s"Peer $id not in seedlist")
```

Comment:
I think `PeerNotOnSeedlist` is the correct wording.

Reply:
👍🏼